integration-tests/js-compute/replace-host.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -lt 2 ]; then
+  cat <<EOF
+Usage: $0 <test-fixture> <override_host_and_protocol>
+
+Replace the host and protocol part of each backend's url with another one. This
+will only replace instances of "JS_COMPUTE_TEST_BACKEND", as other urls might be
+meaningful for an individual test.
+EOF
+  exit 1
+fi
+
+fixtures_dir="$(dirname "${BASH_SOURCE[0]}")/fixtures"
+
+fastly_toml="$fixtures_dir/$1/fastly.toml"
+override_host="$2"
+
+sed -i'' -e "s|JS_COMPUTE_TEST_BACKEND|$override_host|" "$fastly_toml"
```

Comment:
I think it's probably necessary to rename all the `fastly.toml` files to something like `fastly.toml.in`, rather than rewriting them in-place. Eventually we'll run most tests at least twice in CI (once on Viceroy, once on C@E), and I also want people to be able to run the tests locally as many times as they need to. So the act of running the test needs to leave its inputs unchanged so they can be used again.

After doing that, I'd also want `fastly.toml` listed in `integration-tests/js-compute/fixtures/.gitignore`.

If you do rewrite them in-place, a minor nit: `sed -i''` is the same as `sed -i`; the quotes are removed by the shell.

Reply:
Yes, good point. I like explicitly rewriting `fastly.toml.in` :+1: